src/librustc_metadata/encoder.rs

```diff
@@ -1129,31 +1101,32 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
                 }
             }
             None
-        })), self.lazy_seq_ref(&tcx.lang_items.missing))
+        })),
+         self.lazy_seq_ref(&tcx.lang_items.missing))
     }
 
     fn encode_native_libraries(&mut self) -> LazySeq<(NativeLibraryKind, String)> {
         let used_libraries = self.tcx.sess.cstore.used_libraries();
         self.lazy_seq(used_libraries.into_iter().filter_map(|(lib, kind)| {
             match kind {
                 cstore::NativeStatic => None, // these libraries are not propagated
-                cstore::NativeFramework | cstore::NativeUnknown => {
-                    Some((kind, lib))
-                }
+                cstore::NativeFramework | cstore::NativeUnknown => Some((kind, lib)),
             }
         }))
     }
 
     fn encode_codemap(&mut self) -> LazySeq<syntax_pos::FileMap> {
         let codemap = self.tcx.sess.codemap();
         let all_filemaps = codemap.files.borrow();
-        self.lazy_seq_ref(all_filemaps.iter().filter(|filemap| {
-            // No need to export empty filemaps, as they can't contain spans
-            // that need translation.
-            // Also no need to re-export imported filemaps, as any downstream
-            // crate will import them from their original source.
-            !filemap.lines.borrow().is_empty() && !filemap.is_imported()
-        }).map(|filemap| &**filemap))
+        self.lazy_seq_ref(all_filemaps.iter()
+            .filter(|filemap| {
+                // No need to export empty filemaps, as they can't contain spans
+                // that need translation.
+                // Also no need to re-export imported filemaps, as any downstream
+                // crate will import them from their original source.
+                !filemap.lines.borrow().is_empty() && !filemap.is_imported()
+            })
+            .map(|filemap| &**filemap))
```

Comment:
I think this is somewhere we can teach Rustfmt to be a bit smarter, but its not the end of the world IMO. More annoying to me is that the alignment of dots is misleading about the scope.